src/celutil/winutil.cpp

```diff
@@ -10,71 +10,74 @@
 // as published by the Free Software Foundation; either version 2
 // of the License, or (at your option) any later version.
 
-#include <windows.h>
 #include "winutil.h"
 
-using namespace std;
+#include <Windows.h>
```

Comment:
this will not work with mingw crosscompiler
```suggestion
#include <windows.h>
```

Reply:
We have a mixture of `<windows.h>` and `Windows.h` in the source code, so MinGW cross-compilation doesn't seem to be something we support.

Reply:
I even posted brief manual for this - https://github.com/CelestiaProject/Celestia/wiki/crosscompiling-using-mingw. Maybe it's quite outdated